src/index.tsx

```diff
@@ -43,24 +43,30 @@ export const getBasicTemplate = (
   props: BasicTemplateProps,
   options?: Mjml2HtmlOptions
 ) => {
-  const { html } = render(<BasicTemplate {...props} />, options);
-  return html;
+  const { html: body } = render(<BasicTemplate {...props} />, options);
+  const { subject } = props;
+  return { subject, body };
 };
 
 export const getBasicWithAvatarTemplate = (
   props: BasicWithAvatarTemplateProps,
   options?: Mjml2HtmlOptions
 ) => {
-  const { html } = render(<BasicWithAvatarTemplate {...props} />, options);
-  return html;
+  const { html: body } = render(
+    <BasicWithAvatarTemplate {...props} />,
+    options
+  );
+  const { subject } = props;
+  return { subject, body };
 };
 
 export const getResetPasswordTemplate = (
   props: ResetPasswordTemplateProps,
   options?: Mjml2HtmlOptions
 ) => {
-  const { html } = render(<ResetPasswordTemplate {...props} />, options);
-  return html;
+  const { html: body } = render(<ResetPasswordTemplate {...props} />, options);
+  const { subject } = props;
```

Comment:
we can provide default subject for reset password template
```suggestion
  const intl = initIntl();
  const { subject = intl.formatMessage({ id: 'reset-password.subject' }) } = props;
```